drivers/hmis/app/graphql/mutations/add_household_members_to_enrollment.rb

```diff
@@ -13,7 +13,7 @@ class AddHouseholdMembersToEnrollment < BaseMutation
 
     field :enrollments, [Types::HmisSchema::Enrollment], null: true
 
-    def resolve(household_id:, entry_date:, household_members:, confirmed:)
+    def resolve(household_id:, entry_date:, household_members:, confirmed: false)
```

Comment:
This was crashing on the frontend when I merged release-71. Since the argument it optional, we should provide an initializer otherwise it will throw a 500 if the frontend doesn't provide that argument.

Reply:
thanks